src/components/DriverOverview/DriverOverview.tsx

```diff
@@ -0,0 +1,208 @@
+"use client";
+
+import React from "react";
+import { Text, Document, Page, View, StyleSheet, Image } from "@react-pdf/renderer";
+
+export interface DriverOverviewCardProps {
+    driverName: string;
+    date: Date;
+    data: DriverOverviewTableData[];
+}
+
+export interface DriverOverviewTableData {
+    name: string;
+    address: {
+        line1: string;
+        line2: string;
+        town: string;
+        county: string;
+        postcode: string;
+    };
+    contact?: string;
+    packingDate: Date;
+    parcels: number | "None";
+    instructions?: string;
+}
+
+const styles = StyleSheet.create({
+    container: {
+        padding: 25,
+        alignItems: "center",
+    },
+    infoAndLogoContainer: {
+        height: "100",
+        width: "100%",
+    },
+    informationContainer: {
+        border: "2 solid black",
+        alignSelf: "flex-start",
+        margin: 10,
+        padding: 10,
+    },
+    h1text: {
+        fontSize: 24,
+        paddingLeft: 10,
+        paddingRight: 20,
+        paddingBottom: 5,
+        fontFamily: "Helvetica",
+    },
+    h2text: {
+        fontSize: 12,
+        paddingLeft: 10,
+        fontFamily: "Helvetica",
+    },
+    logoStyling: {
+        maxHeight: 60,
+        maxWidth: 60 * 1.7,
```

Comment:
where does 1.7 come from?

Reply:
The aspect ratio for the logo. Will add a comment saying this

Reply:
I think I would rather this just be
`maxWidth: 102, // maintains aspect ratio of logo`